src/main/java/org/folio/rest/exception/HttpException.java

```diff
@@ -17,8 +21,9 @@ public class HttpException extends RuntimeException {
   public HttpException(int code, String message) {
     super(StringUtils.isNotEmpty(message) ? message : ErrorCodes.GENERIC_ERROR_CODE.getDescription());
     this.code = code;
+    var ec = code == 409 ? CONFLICT : GENERIC_ERROR_CODE;
```

Comment:
let's use the same logic for all 3 methods, because in future it can be hard to understand why this method has this logic but 2 other similar does not:

- HttpException(int code, String message)

- HttpException(int code, String message, Throwable cause)

- HttpException(int code, Throwable cause)

Reply:
thanks, Updated them